src/Workspaces/Core/Portable/FindSymbols/FindReferences/DependentTypeFinder.cs

```diff
@@ -177,7 +178,7 @@ internal static class DependentTypeFinder
                     type,
                     solution,
                     null,
-                    (t1, t2) => t1.BaseType == t2,
+                    (candidate, baseType) => AbstractReferenceFinder<INamedTypeSymbol>.OriginalSymbolsMatch(candidate.BaseType, baseType, solution, cancellationToken),
```

Comment:
Do you mean rather than calling `AbstractReferenceFinder<>.OriginalSymbolsMatch()` this gets piped through `SomeOtherHelper.DoSymbolsMatch()` (which eventually calls `OriginalSymbolsMatch`)?